ui/intermittent-failures/App.jsx

```diff
@@ -57,4 +58,4 @@ class App extends React.Component {
   }
 }
 
-export default App;
+export default hot(module)(App);
```

Comment:
`react-hot-loader` requires both handling in our source, and via the webpack configuration. Neutrino 9 has updated from react-hot-loader v3 to v4 on the webpack side, which means we have to do so at the same time. See the migration guide here:
https://github.com/gaearon/react-hot-loader#migrating-from-v3